sub_topic_generator/wikipedia_sub_topic_generation.py

```diff
@@ -3,16 +3,20 @@
 KEY_PHRASES = ["concept", "application", "fundamentals", "principles"]
 
 
-def fuzzy_match(raw_category):
-    category = raw_category.lower()
+def fuzzy_match(search_string):
+    """
+    Determine if raw_category contains any of the words in list KEY_PHRASES
+    :param search_string: str to search and determin if contains KEY_PHRASES
+    :return: bool True if match, False if no match
+    """
+    search_string = search_string.lower()
     for phrase in KEY_PHRASES:
-        if phrase in category:
+        if phrase in search_string:
             return True
-    if category + " stub" in category:
-        return True
+    return False
 
 
-# unused function to print categories that fuzzy match key_phrases
+# TODO: work on this fuction
```

Comment:
@sgmotamedi  and @david-szy5 you can do this in Intellij and it will help you keep track of things you want to do later